app/policies/exercise_policy.rb

```diff
@@ -12,7 +12,7 @@ def show?
     @user.internal_user?
   end
 
-  [:clone?, :destroy?, :edit?, :statistics?, :update?, :feedback?].each do |action|
```

Comment:
Why did you delete the feedback here? We suspect this went wrong during the merge?

Reply:
We still need an answer here